src/java/org/apache/cassandra/tools/nodetool/GetCoerceReadConsistencyAllKeyspaces.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.tools.nodetool;
+
+import io.airlift.command.Command;
+import org.apache.cassandra.tools.NodeProbe;
+import org.apache.cassandra.tools.NodeTool;
+
+@Command(name = "getcoercereadconsistencyallkeyspaces", description = "Get the coerce read consistency level, if present")
+public class GetCoerceReadConsistencyAllKeyspaces extends NodeTool.NodeToolCmd
```

Comment:
Depending on how easy the plumbing is, wanna add an optional `--keyspace` fields that focuses on whether a single keyspace is coerced at all?

Reply:
Added, seems like a pretty minor change, so probably worth it.